libparsec/crates/testbed/src/template/crc_hash.rs

```diff
@@ -173,6 +173,7 @@ macro_rules! impl_crc_hash_for_uuid_based {
 }
 
 impl_crc_hash_for_uuid_based!(VlobID);
+impl_crc_hash_for_uuid_based!(ShamirRevealToken);
```

Comment:
I'm not sure about that. I did that because it was in the 2.17 version

Reply:
See how it's done for `InvitationToken` (it's defined few line below it)